Doc generation consistently fails

The issue at hand seems to be with DDOX:
```
ddox is not present, getting and storing it user wide
No package ddox was found matching the dependency >=0.0.0
##[error]Process completed with exit code 2.
```

Note that we also have quite a few warnings being generated, which could use a fix:
```
Running pre-build commands...
First build / new source files added: Doing a full build...
Type 'Node' implements more than one interface: make sure the one describing the REST server is the first one
source/agora/common/Deserializer.d(126,10): Warning: Ddoc: function declaration has no parameter 'compact'
source/agora/consensus/data/Enrollment.d(36,17): Warning: Ddoc: Stray '('. This may cause incorrect Ddoc output. Use $(LPAREN) instead for unpaired left parentheses.
submodules/ocean/src/ocean/core/Enforce.d(142,13): Warning: Ddoc: function declaration has no parameter 'E'
submodules/ocean/src/ocean/core/Enforce.d(142,13): Warning: Ddoc: parameter count mismatch, expected 5, got 6
submodules/ocean/src/ocean/core/Enforce.d(405,13): Warning: Ddoc: function declaration has no parameter 'E'
submodules/ocean/src/ocean/core/Enforce.d(405,13): Warning: Ddoc: parameter count mismatch, expected 5, got 6
submodules/ocean/src/ocean/core/Test.d(28,1): Warning: Ddoc: Stray ')'. This may cause incorrect Ddoc output. Use $(RPAREN) instead for unpaired right parentheses.
submodules/ocean/src/ocean/core/Test.d(28,1): Warning: Ddoc: Stray ')'. This may cause incorrect Ddoc output. Use $(RPAREN) instead for unpaired right parentheses.
submodules/ocean/src/ocean/core/Test.d(28,1): Warning: Ddoc: Stray ')'. This may cause incorrect Ddoc output. Use $(RPAREN) instead for unpaired right parentheses.
submodules/ocean/src/ocean/core/Traits.d(262,26): Warning: Ddoc: parameter count mismatch, expected 1, got 0
submodules/ocean/src/ocean/core/Traits.d(262,26):        Note that the format is `param = description`
submodules/ocean/src/ocean/core/Traits.d(285,11): Warning: Ddoc: parameter count mismatch, expected 1, got 0
submodules/ocean/src/ocean/core/Traits.d(285,11):        Note that the format is `param = description`
submodules/ocean/src/ocean/core/TypeConvert.d(206,8): Warning: Ddoc: function declaration has no parameter 'From'
submodules/ocean/src/ocean/core/TypeConvert.d(206,8): Warning: Ddoc: parameter count mismatch, expected 1, got 2
submodules/ocean/src/ocean/core/Verify.d(37,13): Warning: Ddoc: parameter count mismatch, expected 4, got 2
submodules/ocean/src/ocean/core/array/Mutation.d(921,12): Warning: Ddoc: function declaration has no parameter 'array'
submodules/ocean/src/ocean/core/array/Mutation.d(980,12): Warning: Ddoc: function declaration has no parameter 'array'
submodules/ocean/src/ocean/core/array/Mutation.d(1035,13): Warning: Ddoc: function declaration has no parameter 'array'
submodules/ocean/src/ocean/core/array/Search.d(215,8): Warning: Ddoc: function declaration has no parameter 'haystac'
submodules/ocean/src/ocean/core/array/Transformation.d(474,12): Warning: Ddoc: function declaration has no parameter 'src'
submodules/ocean/src/ocean/core/array/Transformation.d(601,12): Warning: Ddoc: function declaration has no parameter 'src'
submodules/ocean/src/ocean/meta/types/Typedef.d(145,21): Warning: Ddoc: parameter count mismatch, expected 1, got 0
submodules/ocean/src/ocean/meta/types/Typedef.d(145,21):        Note that the format is `param = description`
submodules/ocean/src/ocean/time/WallClock.d(41,1): Warning: Ddoc: Stray ')'. This may cause incorrect Ddoc output. Use $(RPAREN) instead for unpaired right parentheses.
submodules/ocean/src/ocean/util/log/Hierarchy.d(306,18): Warning: Ddoc: function declaration has no parameter 'Property'
submodules/ocean/src/ocean/util/log/Hierarchy.d(306,18): Warning: Ddoc: parameter count mismatch, expected 2, got 3
submodules/ocean/src/ocean/util/log/ILogger.d(156,17): Warning: Ddoc: parameter count mismatch, expected 1, got 0
submodules/ocean/src/ocean/util/log/ILogger.d(156,17):        Note that the format is `param = description`
```

--- a/README.md
+++ b/README.md
@@ -63,9 +63,9 @@ cd agora/
 # Initialize and update the list of submodules
 git submodule update --init
 # Build the application
-dub build
+dub build --skip-registry=all
 # Build & run the tests
-dub test
+dub test --skip-registry=all
 ```
 
 ## Running tests

--- a/dub.settings.json
+++ b/dub.settings.json
@@ -1,3 +1,0 @@
-{
-    "skipRegistry": "all"
-}

